dotnet/samples/KernelSyntaxExamples/Example28_ActionPlanner.cs

```diff
@@ -23,9 +23,9 @@ public static async Task RunAsync()
             .Build();
 
         string folder = RepoFiles.SamplePluginsPath();
-        kernel.ImportSemanticPluginFromDirectory(folder, "SummarizePlugin");
-        kernel.ImportSemanticPluginFromDirectory(folder, "WriterPlugin");
-        kernel.ImportSemanticPluginFromDirectory(folder, "FunPlugin");
+        kernel.ImportSemanticFunctionsFromDirectory(folder, "SummarizePlugin");
```

Comment:
Change the function to `Folder` or the variable 😊
```suggestion
        kernel.ImportSemanticFunctionsFromFolder(folder, "SummarizePlugin");
```